content/300-guides/050-database/100-developing-with-prisma-migrate/index.mdx

```diff
@@ -28,6 +21,13 @@ When you are comfortable with using Prisma Migrate in development, consider the
 - [How to use Prisma Migrate as a team](team-development)
 - [Advanced migration techniques](customizing-migrations)
 
+<Admonition type="info">
+
+This guide **does not apply for MongoDB**.<br />
+Instead of `migrate dev`, [`db push`](/concepts/components/prisma-migrate/db-push) is used for [MongoDB](/concepts/database-connectors/mongodb).
+
+</Admonition>
```

Comment:
Point out that Migrate for MongoDB is db push instead of "not supported" warning.